modules/core/portal-bootstrap/system.packages.extra.bnd

```diff
@@ -329,5 +328,6 @@ Provide-Capability:\
 	@lib/com.liferay.petra.sql.dsl.spi.jar,\
 	@lib/com.liferay.petra.string.jar,\
 	@lib/com.liferay.petra.url.pattern.mapper.jar,\
-	@lib/com.liferay.petra.xml.jar
+	@lib/com.liferay.petra.xml.jar,\
+	@lib/com.liferay.portal.encryptor.jar
```

Comment:
@julschong I do not think you need it here, as only implementation included in com.liferay.portal.encryptor.jar, nothing to export. 
Here is a homework for you, figure out how this system.packages.extra.bnd work, slack me if you any trouble understanding it.

Tina.